arangod/Agency/AgencyFeature.cpp

```diff
@@ -95,7 +96,9 @@ void AgencyFeature::collectOptions(std::shared_ptr<ProgramOptions> options) {
   options->addOption(
       "--agency.election-timeout-min",
       "minimum timeout before an agent calls for new election (in seconds)",
-      new DoubleParameter(&_minElectionTimeout),
+      new DoubleParameter(&_minElectionTimeout, /*base*/ 1.0, /*minValue*/ 0.0,
+                          /*maxValue*/ std::numeric_limits<double>::max(),
+                          /*minInclusive*/ false),
```

Comment:
Instead of having unnamed parameter and adding comments since c++20 once can use [designated initializers](https://en.cppreference.com/w/cpp/language/aggregate_initialization#Designated_initializers).

[Godbolt](https://godbolt.org/#g:!((g:!((g:!((h:codeEditor,i:(filename:'1',fontScale:14,fontUsePx:'0',j:1,lang:c%2B%2B,selection:(endColumn:45,endLineNumber:12,positionColumn:45,positionLineNumber:12,selectionStartColumn:45,selectionStartLineNumber:12,startColumn:45,startLineNumber:12),source:'%0Astruct+MyOptions%7B%0A++++double+min%3B%0A++++double+max%3B%0A++++double+value%3B%0A%7D%3B%0A%0A%0Avoid+baz(MyOptions)%3B%0A%0Avoid+foo()+%7B%0A++++baz(%7B.min+%3D+12,+.max+%3D+15,+.value+%3D+7%7D)%3B%0A%7D%0A%0A'),l:'5',n:'0',o:'C%2B%2B+source+%231',t:'0')),k:47.154046997389024,l:'4',n:'0',o:'',s:0,t:'0'),(g:!((g:!((h:compiler,i:(compiler:clang1301,filters:(b:'0',binary:'1',commentOnly:'0',demangle:'0',directives:'0',execute:'0',intel:'0',libraryCode:'0',trim:'1'),flagsViewOpen:'1',fontScale:14,fontUsePx:'0',j:1,lang:c%2B%2B,libs:!(),options:'-std%3Dc%2B%2B20+-O3',selection:(endColumn:1,endLineNumber:1,positionColumn:1,positionLineNumber:1,selectionStartColumn:1,selectionStartLineNumber:1,startColumn:1,startLineNumber:1),source:1,tree:'1'),l:'5',n:'0',o:'x86-64+clang+13.0.1+(C%2B%2B,+Editor+%231,+Compiler+%231)',t:'0')),k:49.63387691952499,l:'4',m:68.17434210526315,n:'0',o:'',s:0,t:'0'),(g:!((h:output,i:(editorid:1,fontScale:14,fontUsePx:'0',j:1,wrap:'1'),l:'5',n:'0',o:'Output+of+x86-64+clang+13.0.1+(Compiler+%231)',t:'0')),header:(),l:'4',m:31.825657894736846,n:'0',o:'',s:0,t:'0')),k:52.84595300261096,l:'3',n:'0',o:'',t:'0')),l:'2',n:'0',o:'',t:'0')),version:4)

Reply:
Agreed to move to designated initializers in a separate step.